mitosheet/mitosheet/steps_manager.py

```diff
@@ -276,7 +276,7 @@ def analysis_data_json(self):
                 "stepSummaryList": self.step_summary_list,
                 "currStepIdx": self.curr_step_idx,
                 "dataTypeInTool": self.data_type_in_mito.value,
-                "graphDataJSON": self.curr_step.graph_data
+                "graphDataJSON": self.curr_step.graph_data_dict
```

Comment:
Should this be named graphDataDict? I think yes; we want to keep these consistent, if we can.